source/extension/georef/examples/gcp-georeferencing.json

```diff
@@ -8,14 +8,17 @@
   ],
   "motivation": "georeferencing",
   "target": {
-    "type": "Image",
-    "source": "https://cdm21033.contentdm.oclc.org/digital/iiif/krt/2891/full/full/0/default.jpg",
-    "service": [
-      {
-        "@id": "https://cdm21033.contentdm.oclc.org/digital/iiif/krt/2891",
-        "type": "ImageService2"
-      }
-    ],
+    "type": "SpecificResource",
+    "source": {
+      "@id" : "https://cdm21033.contentdm.oclc.org/digital/iiif/krt/2891/full/full/0/default.jpg",
```

Comment:
I think this is targeting the Image which might cause confusion with saying you only support IIIF images...

Reply:
I have taken this into account.  This discussion has been turned into prototypes available from some tangential work.  Thank you!